hedera-mirror-test/build.gradle.kts

```
@@ -70,6 +70,8 @@ dependencies {
 tasks.named("test") { enabled = false }
 
 tasks.register<Test>("acceptance") {
+    description = "Acceptance tests configuration"
+    group = "Testing"
```

Since test task is under verification group, it makes sense to also have acceptance there.

```suggestion
    group = "verification"
```

---

Fixed